cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -178,8 +282,617 @@ class GameWorldService {
             });
 
         unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_class",
-            [this](const uint64_t& instance) -> uint64_t {
-                return reinterpret_cast<uint64_t>(reinterpret_cast<UObject*>(instance)->GetClass());
+            [this](const uint64_t& uobject) -> uint64_t {
+                return reinterpret_cast<uint64_t>(reinterpret_cast<UObject*>(uobject)->GetClass());
+            });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_static_class",
+            [this](const std::string& class_name) -> uint64_t {
+                return reinterpret_cast<uint64_t>(UnrealClassRegistrar::getStaticClass(class_name));
+            });
+
+        // TODO: How do we expose FActorSpawnParameters?
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "spawn_actor",
+            [this](const std::string& class_name, const std::vector<double>& location, const std::vector<double>& rotation, const spawn_params) -> uint64_t {
```

Comment:
How does this compile? What is the type of `spawn_params`?

Reply:
aah, my bad, this functionality should have been commented when I pushed the changes.